ENIGMAsystem/SHELL/Graphics_Systems/Direct3D11/DX11textures.cpp

```diff
@@ -141,9 +128,9 @@ void graphics_replace_texture_alpha_from_texture(int tex, int copy_tex)
 
 void graphics_delete_texture(int tex)
 {
-  texture_peers[tex].second->Release();
-  texture_peers[tex].first->Release();
-  texture_peers[tex] = {NULL,NULL}; // null the pointers
+  auto texture = (DX11Texture*)textures[tex];
+  texture->peer->Release(); texture->peer = NULL;
+  texture->view->Release(); texture->view = NULL;
```

Comment:
Don't actually care, but prefer commas to divide statements on a line, and also spaces after casts.

Reply:
Good point, I agree on the first thing, but I don't like adding a space after a cast it's just not my style.

Reply:
Here: 7bc8c87c0652c191294039e5bea6790852eba4d2, 2c54773ebff70385a017e73b7588b9e9be96c543

I like that and from here on out I will be nulling my COM pointers all on the same line. I will also use comma operators to separate statements on the same line since it does not result in ambiguous indentation.